### ISSUE TITLE
C --> Fortran array deep copy

### ISSUE THREAD

**Issue by ercius** [OpenChemistry/tomviz#880]
The console keeps printing out the following information:

> Warning, array does not have Fortran order, making deep copy and fixing...
...done.

This seems very inefficient. I am not sure why tomviz has to keep changing the array from Fortran -->C-->Fortran ordering. This happens for almost any numpy operation since numpy is C ordering.

As a test I modified the rotation script to print out the ndarray flags at various points:
```
print('Rotating Dataset...')
print(data_py.flags) #print the ndarray flags
print('...')
axis1 = (rotation_axis + 1) % 3
axis2 = (rotation_axis + 2) % 3
data_py_return = ndimage.interpolation.rotate(
    data_py, rotation_angle, axes=(axis1, axis2),order=1,mode='nearest')
print(data_py_return.flags) #print the ndarray flags
utils.set_array(dataset, data_py_return)
print('Rotation Complete')
```

Here is the output:
> Rotating Dataset...
  C_CONTIGUOUS : False
  F_CONTIGUOUS : True
  OWNDATA : False
  WRITEABLE : True
  ALIGNED : True
  UPDATEIFCOPY : False
  ...
  C_CONTIGUOUS : True
  F_CONTIGUOUS : False
  OWNDATA : True
  WRITEABLE : True
  ALIGNED : True
  UPDATEIFCOPY : False
Warning, array does not have Fortran order, making deep copy and fixing...
...done.
Rotation Complete

So, the array starts as Fortran and ndarray.interpolation.rotate returns a C-ordering array. Then the utils sets it back to fortran ordering. I guess tomviz internals (based on VTK?) need Fortran ordered arrays? It would be better if we can use 1 ordering to avoid deep copies.

As a side note: Some of the scripts seem to use fortran indexing [X,Y,num] rather than the numpy default C indexing [num,Y,X]. This might be correct initially, but some operations might return a C-ordered array, It is important to be clear of the array ordering during image-by-image operations (such as cross-correlation) because you can get a large speed up with the correct indexing.

**Comment by cjh1** [comment]
@ercius #882 resolves this issue for rotation, but we will have to apply a similar fix else where.

**Comment by cjh1** [comment]
Fixed by #880
